### PR TITLE
Propose owners for JVM/Software platforms

### PR DESCRIPTION
**Pull request by big-guy**

### CODE REVIEW

**Review by nathan-contino — APPROVED** [review]
LGTM. I checked the CODEOWNERS documentation and I couldn't find anything about mutual exclusion, so... we'll just have to see how GitHub handles this. I'd like to think it'll treat us as co-owners.

FYI: if we can refer to folders instead of specific files, that would be much more futureproof than listing specific files (that are subject to name changes that'll be hard to remember to update in CODEOWNERS).

**Comment by big-guy** [comment]
@nathan-contino rgr

RE: files vs folders, I would rather do folders too, but these files are clearly JVM-related and aren't in a JVM-only directory right now.

**Comment by tresat** [comment]
> LGTM. I checked the CODEOWNERS documentation and I couldn't find anything about mutual exclusion, so... we'll just have to see how GitHub handles this. I'd like to think it'll treat us as co-owners.
> 
> FYI: if we can refer to folders instead of specific files, that would be much more futureproof than listing specific files (that are subject to name changes that'll be hard to remember to update in CODEOWNERS).

Sounds like you're volunteering to organize the docs by platform? I think that would be very helpful! 😄

**Review by tresat — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -19,10 +19,53 @@ subprojects/enterprise-logging/     @gradle/ge-build-insights @gradle/ge-testing
...
+subprojects/docs/src/docs/userguide/core-plugins/build_init_plugin.adoc           @gradle/bt-jvm
+subprojects/docs/src/docs/userguide/core-plugins/checkstyle_plugin.adoc           @gradle/bt-jvm
+subprojects/docs/src/docs/userguide/core-plugins/codenarc_plugin.adoc             @gradle/bt-jvm
+subprojects/docs/src/docs/userguide/core-plugins/distribution_plugin.adoc         @gradle/bt-jvm
+subprojects/docs/src/docs/userguide/core-plugins/ear_plugin.adoc                  @gradle/bt-jvm
+subprojects/docs/src/docs/userguide/core-plugins/jacoco_plugin.adoc               @gradle/bt-jvm
+subprojects/docs/src/docs/userguide/core-plugins/java_gradle_plugin.adoc          @gradle/bt-jvm
+subprojects/docs/src/docs/userguide/core-plugins/pmd_plugin.adoc                  @gradle/bt-jvm
+subprojects/docs/src/docs/userguide/core-plugins/war_plugin.adoc                  @gradle/bt-jvm
+subprojects/docs/src/docs/userguide/dep-man/                                      @gradle/bt-jvm
```
Comment (jvandort) [inline .github/CODEOWNERS:68]: I see we're assigning dependency management to our team, as well as assigning the software platform to us as well. In that case, does it make sense to also add `subprojects/dependency-management` to the `#Software Platform` list above? 

Also, we should consider splitting up the docs sections by platform as well.

**Review by jvandort — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (big-guy) [reply .github/CODEOWNERS:68]: > In that case, does it make sense to also add subprojects/dependency-management to the #Software Platform list above?

Maybe, I was trying to treat all of docs as a single thing and just plucking out pieces.

**Review by octylFractal — COMMENTED** [review]
Do we want to add sections of `core`? e.g. `subprojects/core/src/main/java/org/gradle/internal/jvm`?

**Comment by big-guy** [comment]
> Do we want to add sections of `core`? e.g. `subprojects/core/src/main/java/org/gradle/internal/jvm`?

I wouldn't go that far yet. It might be better to bust them up.

**Comment by big-guy** [comment]
@bot-gradle test and merge

**Pull request merged**